docs/web3-eth-subscribe.rst

```diff
@@ -78,7 +78,7 @@ clearSubscriptions
 
 Resets subscriptions.
 
-.. note:: This will not reset subscriptions from other packages like ``web3-shh``, as they use their own requestManager.
+.. note:: This will not reset subscriptions from other packages like.
```

Comment:
No longer a complete sentence, either give another package example, or even remove the line?